### PR TITLE
zed: send notification email by default

### PR DESCRIPTION
**Pull request by shodanshok**
### Motivation and Context

Important notification (ie: a failed disk or a degraded pool) should be sent to root by default.

### Description

Simply enable email notification by default, un-commenting `ZED_EMAIL_ADDR="root"`

### How Has This Been Tested?

The above line is un-commented on all the machines I manage.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Performance enhancement (non-breaking change which improves efficiency)
- [ ] Code cleanup (non-breaking change which makes code smaller or more readable)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Library ABI change (libzfs, libzfs\_core, libnvpair, libuutil and libzfsbootenv)
- [ ] Documentation (a change to man pages or other documentation)

### Checklist:

- [x] My code follows the OpenZFS [code style requirements](https://github.com/openzfs/zfs/blob/master/.github/CONTRIBUTING.md#coding-conventions).
- [x] I have updated the documentation accordingly.
- [x] I have read the [**contributing** document](https://github.com/openzfs/zfs/blob/master/.github/CONTRIBUTING.md).
- [ ] I have added [tests](https://github.com/openzfs/zfs/tree/master/tests) to cover my changes.
- [ ] I have run the ZFS Test Suite with this change applied.
- [x] All commit messages are properly formatted and contain [`Signed-off-by`](https://github.com/openzfs/zfs/blob/master/.github/CONTRIBUTING.md#signed-off-by).

### CODE REVIEW

**Review by gmelikov — APPROVED** [review]

**Comment by gyakovlev** [comment]
this is the first change I do after installing zfs on new systems, agreed 👍

File: cmd/zed/zed.d/zed.rc [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # Email will only be sent if ZED_EMAIL_ADDR is defined.
 # Disabled by default; uncomment to enable.
```
Comment (behlendorf) [inline cmd/zed/zed.d/zed.rc:16]:
```suggestion
# Enabled by default; comment to disable.
```

**Review by behlendorf — COMMENTED** [review]
It seems like the least surprising default behavior would be to leave this disabled by default.  For those who want it, it's easy enough to enable.  But I'm willing to be convinced, are there other common packages which enable this kind of email notification by default?

File: cmd/zed/zed.d/zed.rc (hunk 1 above)
Comment (shodanshok) [reply cmd/zed/zed.d/zed.rc:16]: @behlendorf thanks for catching the comment, I totally missed it.

**Comment by shodanshok** [comment]
> It seems like the least surprising default behavior would be to leave this disabled by default. For those who want it, it's easy enough to enable. But I'm willing to be convinced, are there other common packages which enable this kind of email notification by default?

@behlendorf  On both RedHat/CentOS and Debian/Ubuntu, `md` does monitor array status writing to `root` if anything bad happens. The same can be said for `smartd`, which alerts if some critical parameters (ie: reallocated sector count) changes.

The rationale is simple: disks keep user data, and the user should be alerted if data are at risk.

By default, `zed` does not alert for harmless events (ie: a scrub completed with no issue), so I can't find anything bad in enabling reporting for potentially very dangerous events.

**Comment by bghira** [comment]
FreeBSD also has daily anacron emails that go out by default.

**Review by behlendorf — APPROVED** [review]
That's good enough for me.  Thanks for the examples.

**Comment by behlendorf** [comment]
@shodanshok I've merged the suggested comment change in to this PR.  If you could just add your signed-off-by to the commit and update the PR one last time it should be good to go.

**Review by gmelikov — APPROVED** [review]

**Comment by shodanshok** [comment]
@behlendorf thank you and George for accepting the proposed change. I've updated the PR with a new Signed-Off-By commit; I hope having done that as required (if not, please instruct me on what/how to do it the correct way).

**Comment by behlendorf** [comment]
That'll work.  Thanks, I've merged it.

**Pull request merged**